include/measurement_kit/ooni/error.hpp

```diff
@@ -9,23 +9,22 @@
 namespace mk {
 namespace ooni {
 
-MK_DEFINE_ERR(MK_ERR_OONI(0), CannotOpenInputFileError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(1), MissingRequiredInputFileError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(2), MissingCollectorBaseUrlError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(3), CannotOpenReportError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(4), MissingMandatoryKeyError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(5), InvalidMandatoryValueError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(6), MissingRequiredHostError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(7), MissingRequiredUrlError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(8), GeoipDatabaseOpenError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(9), GeoipCountryCodeLookupError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(10), GeoipCountryNameLookupError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(11), GeoipCityLookupError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(12), GeoipAsnLookupError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(13), CannotGetResourcesVersionError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(14), CannotGetResourcesManifestError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(15), CannotGetResourceError, "")
-MK_DEFINE_ERR(MK_ERR_OONI(16), ResourceIntegrityError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(0), MissingRequiredInputFileError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(1), MissingCollectorBaseUrlError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(2), CannotOpenReportError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(3), MissingMandatoryKeyError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(4), InvalidMandatoryValueError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(5), MissingRequiredHostError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(6), MissingRequiredUrlError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(7), GeoipDatabaseOpenError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(8), GeoipCountryCodeLookupError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(9), GeoipCountryNameLookupError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(10), GeoipCityLookupError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(11), GeoipAsnLookupError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(12), CannotGetResourcesVersionError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(13), CannotGetResourcesManifestError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(14), CannotGetResourceError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(15), ResourceIntegrityError, "")
```

Comment:
Why did the error codes change numbers?

I think you did it because you removed the "CannotOpenInputFileError", but isn't it better to perhaps keep the error codes consistent and just leave a "hole" where that error once was?